Rename "CI Integrations" to "Notifications" (#191)
returntocorp/semgrep-docs: docs/notifications.md

![A correctly configured Slack webhook will send a notification like this](img/test-notification.png "Correctly configured webhook will send a notification like this")

## Email

To receive email notifications about Semgrep findings on pull requests and code pushes, visit [Dashboard > Integrations](https://semgrep.dev/manage/integrations) and select 'Add integration' or 'Setup First Integration,' and then choose 'Email'. Enter your email address, give the channel a name of your choosing, and then click 'Save'.

On each scan that has at least one finding, you will receive one email from Semgrep with a summary of all of the findings from that scan.

## GitHub pull request comments

> !!! info

Reviewer: Do we still need this info box given that the subheading directly above already includes “GitHub?”

Author: good point!

- `SEMGREP_PR_ID` is set to the PR number of the pull request on Github (e.g. `2900`)
- `SEMGREP_REPO_NAME` is set to the repo name (e.g. `returntocorp/semgrep`)

## GitLab merge request comments

> Merge request comments on GitLab are coming soon. [Sign up for the beta here.](https://go.r2c.dev/join-gitlab-beta)

Reviewer: ```suggestion
Interested in GitLab merge request comments? [Sign up for the beta here.](https://go.r2c.dev/join-gitlab-beta)
```

## Webhooks

> Webhook notifications are a paid feature in our Semgrep Team tier.

Reviewer: ```suggestion
Webhook notifications are a paid feature in the Semgrep Team tier.
```
